### DISCUSSION TITLE
Point instancer overrides.

### DISCUSSION THREAD

**Discussion by daviddeacon** [Autodesk/maya-usd#2135] [Q&A] [answered]
Hi,

I'm not sure if this is a USD limitation or a limitation of the Maya USD plugin.

I have a forest set usd file that has a bunch of trees in a point instancer. It's great that I can still tweak their positions on a shot level, but I've found that the override that's written to the shot usd file contains positions for all of the unedited point instances. This would block any updates to the forest set usd from propagating. 

Is it possible to have the point positions either written in a sparse manor, or written as relative transforms vs absolute?

David.

**Comment by kxl-adsk** [comment, marked as answer]
Hi @daviddeacon,

This is a USD limitation - it doesn't support sparse array overrides. Spiff outlined some of the challenges in getting this to work in [this thread](https://groups.google.com/g/usd-interest/c/dHGzbJeiRRA/m/XAEHPTgICQAJ).

Cheers,
KxL

**Comment by daviddeacon** [comment]
Ah, I had a feeling this might be the answer.

Thanks anyway.
David.
